libcaf_io/test/basp_udp.cpp

```diff
@@ -685,7 +701,7 @@ CAF_TEST(publish_and_connect_udp) {
 
 CAF_TEST(remote_actor_and_send_udp) {
   constexpr const char* lo = "localhost";
-  CAF_MESSAGE("self: " << to_string(self()->address()));
+  CAF_MESSAGE("self:    " << to_string(self()->address()));
```

Comment:
Please don't add whitespaces to the output. You probably tried to align some of the output, but it's simply not worthwhile doing this by hand.